apdet-hanna/src/apdet.cpp

```diff
@@ -393,13 +392,12 @@ bool nearingApogee(int16_t accel, float base_alt, float curr_pres)
   * @param  height      The last recorded height.
   * @return Boolean
   */
-bool testApogee(float base_alt, float curr_pres, float *height)
+bool testApogee(float base_alt, float curr_pres, float *prev_height, float height)
 {
+    bool apogee = ((height - *prev_height) <= 0);
```

Comment:
I'd have named this variable something like "detected_apogee", but that's just me